snapshot/restore/restore.go

```diff
@@ -66,6 +67,7 @@ type Options struct {
 	Parallel     int  `json:"parallel"`
 	Incremental  bool `json:"incremental"`
 	IgnoreErrors bool `json:"ignoreErrors"`
+	MaxDepth     int  `json:"maxdepth"`
```

Comment:
nit: `maxDepth` 

also, perhaps call this `RestoreDirEntryAtDepth` or something similar?

I imagine we can decide to restore some files as placeholders as well (perhaps above certain sice, etc.) is that right?

Reply:
Rename done.

Yes. There's a TODO about directly restoring files below a certain (configurable) size. I'd prefer to add this in a subsequent PR. WDYT?